docs/reference/builder.md

```diff
@@ -1754,6 +1754,6 @@ FROM ubuntu
 RUN echo moo > oink
 # Will output something like ===> 695d7793cbe4
 
-# You᾿ll now have two images, 907ad6c2736f with /bar, and 695d7793cbe4 with
+# You'll now have two images, 907ad6c2736f with /bar, and 695d7793cbe4 with
```

Comment:
Looks like this is also in this PR https://github.com/docker/docker/pull/29583/files#diff-4fc21624aa5dadd153e2780910da7c7dR1757